projects/arlas-components/assets/i18n/es.json

```diff
@@ -0,0 +1,95 @@
+{
+	"BBox generator": "Generar un cuadro delimitador",
+	"First corner": "Primera esquina",
+	"Second corner": "Segunda esquina",
+	"Close": "Cerrar detalle",
```

Comment:
Are we sure the close key is only used for details ?

Reply:
fixed